### PR TITLE
blacklist specific omnifunc also for CSS and JS

### PR DESCRIPTION
**Pull request by loomchild**

### CODE REVIEW

File: rplugin/python3/deoplete/source/omni.py [hunk 1]
```diff
@@ -61,11 +61,13 @@ def _get_complete_position(self, context, current_ft, filetype):
                                            'Manual' and m is None):
                     continue
 
-                if filetype == current_ft and self._omnifunc in [
+                if ((filetype == current_ft or
+                        filetype in ['css', 'javascript'])
```
Comment (loomchild) [inline rplugin/python3/deoplete/source/omni.py]: Alternatively remove filetype condition (remove `((filetype == current_ft or filetype in ['css', 'javascript'])`) and rely only on blacklisted omnifunc's.

Comment (Shougo) [reply rplugin/python3/deoplete/source/omni.py]: I think specify omnifuncs names are better.

**Comment by Shougo** [comment]
Why the feature is needed?   Please describe it.

**Comment by Shougo** [comment]
Oh, OK.
It is for #914.

**Pull request closed without merging**